vck/src/commonMain/kotlin/at/asitplus/wallet/fallbackCredential/vcFallbackCredentialScheme/VcFallbackCredentialScheme.kt

```diff
@@ -0,0 +1,13 @@
+package at.asitplus.wallet.fallbackCredential.vcFallbackCredentialScheme
+
+import at.asitplus.wallet.lib.data.ConstantIndex
+
+data class VcFallbackCredentialScheme(override val vcType: String? = null) : ConstantIndex.CredentialScheme {
```

Comment:
`vcType` should not be null in this class, right? Because otherwise we wouldn't instantiate it.